src/recipes/raspberry-caipirinha.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "Raspberry Caipirinha",
+    "description": "Traditional Caipirinha but with the sweetness of the strawberry",
+    "github": "Mayie",
+    "ingredients": [
```

Comment:
All ingredient itens indentation isn't correct. Add a tab/space for all block and remove all empty lines,